.github/workflows/docker_build.yml

```diff
@@ -98,4 +102,162 @@ jobs:
         file: ./Dockerfile.jdk8
         push: true
         tags: hyperledgerlabs/baf-build:jdk8
+  cordaOS-latest: 
+    if: ${{ startsWith(github.ref, 'refs/heads/develop') }}
+    runs-on: ubuntu-latest
+    steps:
+    - 
+      name: Checkout
+      uses: actions/checkout@v2
+    - 
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'adopt'
+        java-version: '8'
+    - name: Set output
+      id: vars
+      run: echo ::set-output name=tag::${GITHUB_REF#refs/tags/v}
+    -
+      name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+    -
+      name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    -
+      name: Login to DockerHub
+      uses: docker/login-action@v1 
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+    - 
+      name: corda builds
+      run: |
+        WDIR=`pwd`
+        
+        java -version
+        update-alternatives --list java
+
+        echo "---- building networkmap ----"
+        cd ${WDIR}/platforms/r3-corda/images/networkmap
+        mvn clean install -DskipTests   
+
+        echo "---- building doorman ----"
+        cd ${WDIR}/platforms/r3-corda/images/doorman
+        mvn clean install -DskipTests  
+    -
+      name: Build and push cordaOS doorman latest  
+      id: corda_doorman_latest
+      uses: docker/build-push-action@v2
+      with:
+        context: platforms/r3-corda/images/doorman
+        push: true
+        tags: hyperledgerlabs/doorman-linuxkit:latest            
+    -
+      name: Build and push cordaOS networkmap latest
+      id: corda_networkmap_latest
+      uses: docker/build-push-action@v2
+      with:
+        context: platforms/r3-corda/images/networkmap
+        push: true
+        tags: hyperledgerlabs/networkmap-linuxkit:latest    
+    -
+      name: Build and push cordaOS node latest 
+      id: corda_node_latest
+      uses: docker/build-push-action@v2
+      with:
+        file: platforms/r3-corda/images/Dockerfile-corda-tls
+        push: true
+        tags: hyperledgerlabs/corda-linuxkit:latest 
```

Comment:
we can't have image like corda-linuxkit, it has to be corda:linuxkit-latest or just latest, lets remove linuxkit from the tags